site2/docs/cookbooks-encryption.md

```diff
@@ -1,334 +0,0 @@
----
-id: cookbooks-encryption
-title: Configure end-to-end encryption
-sidebar_label: "Configure end-to-end encryption"
----
-
-````mdx-code-block
```

Comment:
It's possible to set up a client-side redirection here:

```jsx
import {Redirect} from '@docusaurus/router';

<Redirect to="security-encryption#get-started" />
```

So that users who have a bookmark to the original page won't get a 404 but be redirected to the new section. However, it's also reasonable to just remove it since I believe few users will depend on this page. Just for your information.

Reply:
@tisonkun Thanks for the information. Will give it a try and see.